ansible_base/lib/abstract_models/common.py

```diff
@@ -57,7 +57,7 @@ class Meta:
 
     def _attributable_user(self, warn_nonexistent_system_user):
         user = get_current_user()
-        if user is None:
+        if user is None or user.is_anonymous:
```

Comment:
Isn't it very concerning if an object is modified by an anonymous user? Wouldn't it be good to log that?

Reply:
Yea, this is the debate because we have one spot where we know this happens. When a user is logging in from an authenticator they are AnonymousUser when it tries to save the new User for them. I am going to push a change where I try to better catch just that exception and error otherwise.